File: app/src/main/java/com/adityaarora/liveedgedetection/DetectApplication.java
package com.adityaarora.liveedgedetection;

import android.app.Application;

import info.hannes.crashlytic.CrashlyticsTree;
import info.hannes.timber.DebugTree;
import timber.log.Timber;

public class DetectApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        Timber.plant(new DebugTree());
        if (!BuildConfig.DEBUG)
            Timber.plant(new CrashlyticsTree());
    }
}

File: app/src/main/java/com/adityaarora/liveedgedetection/DetectApplication.kt
package com.adityaarora.liveedgedetection

import android.app.Application
import info.hannes.crashlytic.CrashlyticsTree
import info.hannes.timber.DebugTree
import timber.log.Timber

class DetectApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Timber.plant(DebugTree())
        if (!BuildConfig.DEBUG) Timber.plant(CrashlyticsTree())
    }
}